generate_admin_token.py

```diff
@@ -11,14 +14,10 @@ def print_usage_and_exit():
 def main(key):
     '''
     '''
-    url = 'http://localhost:8080/secret_key'
-    values = { 'key_string': key }
-    data = urllib.urlencode(values)
-    request = urllib2.Request(url, data)
-    response = urllib2.urlopen(request)
+    uuidgen > app/handlers/admin_uuid.txt
```

Comment:
It is not used in the docker version of stochss.  Please don't remove it yet, as it is used in other deployments.  We need to unify the way these work, which could mean using this file still.